### PR TITLE
add columns dst_country_name, src_network_name and dst_network_name

### PR DESCRIPTION
**Pull request by Ivanov-Anton**
Added additional virtual fields for Cdr model

Screenshots:
----------------------------------------

### CODE REVIEW

**Comment by dmitry-sinina** [comment]
Currently we have src_country_id, dst_country_id, src_network_id, dst_network_id. Could you add src_country_name, dst_country_name, src_network_name and dst_network_name

**Comment by Fivell** [comment]
@Ivanov-Anton

**Comment by Ivanov-Anton** [comment]
OK

File: app/admin/cdr/cdr_exports.rb [hunk 1]
```diff
@@ -74,11 +74,14 @@
   end
 
   member_action :download do
-    response.headers['X-Accel-Redirect'] = "/x-redirect/cdr_export/#{resource.id}.csv"
-    response.headers['Content-Type'] = 'text/csv; charset=utf-8'
-    response.headers['Content-Disposition'] = "attachment; filename=\"#{resource.id}.csv\""
-
-    render body: nil
+    if resource.status == CdrExport::STATUS_COMPLETED
+      file_name = "#{Rails.configuration.yeti_web.fetch('cdr_export').fetch('dir_path')}/#{resource.id}.csv"
```
Comment (Fivell) [inline app/admin/cdr/cdr_exports.rb:78]: this should work with X-Accel-Redirect, file can exist on different server

File: app/models/cdr_export.rb [hunk 2]
```diff
@@ -91,7 +91,45 @@ def fields=(f)
...
-    Cdr::Cdr.select(fields.join(', ')).order('time_start desc').ransack(filters.as_json).result.to_sql
+    s = Cdr::Cdr.select(select_sql)
+    s = s.order('time_start desc')
+    s = s.ransack(filters.as_json).result
+    if fields.include?('src_country_name')
+      s = s.joins('LEFT JOIN external_data.countries as src_c ON cdr.cdr.src_country_id = src_c.id')
+    end
+    if fields.include?('dst_country_name')
+      s = s.joins('LEFT JOIN external_data.countries as dst_c ON cdr.cdr.dst_country_id = dst_c.id')
+    end
```
Comment (gigorok) [inline app/models/cdr_export.rb:102]: no needs to join 2 times when 2 filters by same table were passed

**Pull request merged**